src/style.css

```diff
@@ -0,0 +1,50 @@
+/* Add styles */
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+h1 {
+  font-size: 25px;
+  text-align: center;
+}
+
+.refresh-scores {
+  display: flex;
+  justify-content: space-between;
+}
+
+.refresh-scores h2 {
+  font-size: 20px;
+}
+
+#refresh-button {
+  margin: 18px;
+}
+
+#scores-display {
+  border: 3px solid black;
+  padding: 5px 1px;
+}
+
+#scores-display div:nth-of-type(2n) {
```

Comment:
- [ ] Please don't use id's as selectors. This is because IDs can only be used once on a web page. With an ID, on the other hand, we could not replicate this behavior, because each element can only have one ID.